lib/assembly/application.ex

```diff
@@ -7,16 +7,35 @@ defmodule Assembly.Application do
 
   require Logger
 
+  import Cachex.Spec
+  import Supervisor.Spec
+
   def start(_type, _args) do
     children = [
+      {DynamicSupervisor, name: Assembly.BuildSupervisor, strategy: :one_for_one},
+      Assembly.Repo,
+      {Cachex, cachex_opts()},
+      supervisor(GRPC.Server.Supervisor, [{Assembly.Endpoint, 50_051}]),
       {Assembly.Broadway, []}
     ]
 
     Logger.info("Starting Assembly")
 
-    # See https://hexdocs.pm/elixir/Supervisor.html
-    # for other strategies and supported options
     opts = [strategy: :one_for_one, name: Assembly.Supervisor]
     Supervisor.start_link(children, opts)
   end
+
+  defp cachex_opts do
```

Comment:
I'm not in love with this but without it Cachex attempts to trigger the warmer on test, since we're using our mock and there is no expectation defined yet this errors.